mod_deploy/controllers.py

```diff
@@ -63,6 +63,9 @@ def decorated_function(*args, **kwargs):
 
 
 def is_valid_signature(x_hub_signature, data, private_key):
+    """
+    Re-check if the GitHub hook request got valid signature
```

Comment:
Please make sure the doc-blocks fullfil the PEP-257, as listed here: https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md#requirements-for-a-pull-request

Reply:
for all doc-blocks right?

Reply:
Besides this, please also add descriptions for the types and arguments (see https://github.com/CCExtractor/sample-platform/blob/master/config_parser.py of an example)

Reply:
Yes, all added doc-blocks need to conform to that PEP.